src/main/java/no/unit/nva/bare/FetchAuthorityHandler.java

```diff
@@ -58,11 +61,11 @@ public GatewayResponse handleRequest(final Map<String, Object> input, Context co
                     BareAuthority fetchedAuthority = bareConnection.get(arpId);
```

Comment:
Splitt opp

Reply:
I have created a task for upgrading this handler (outside scope of task at hand, but changes in dependencies forced a few changes nonetheless)

https://unit.atlassian.net/browse/NP-1063